src/main/kotlin/io/github/anki/anki/service/CardsService.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.anki.anki.service
+
+import io.github.anki.anki.controller.model.CardDto
+import io.github.anki.anki.repository.mongodb.CardRepository
+import io.github.anki.anki.repository.mongodb.model.MongoCard
+import io.github.anki.anki.service.model.Card
+import org.bson.types.ObjectId
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import java.util.*
+
+
+@Service
+class CardsService(
+    private val dataSource: CardRepository) {
+
+    fun getCards(collectionId: String): List<Card> {
```

Comment:
How should it works? The no any collection model present in your PR. 
I highly recommend implement functionality with relation.

### in the next PR
ONLY if one card can be associated only to one collection, then   
1a. Add to Card field named `collectionId`
Otherwise simplified way can be like this
1b. Add to Card field array named collections, that will store all collections related to this card
But if you want to search cards by collections you probably won't to iterate by all cards collection to find all related cards, then 
1c. Remove all information about collection from Card and create additional collection that can be named CardsCollections and store relations between cards and collections here. The search by collection in this case would be much faster compared to 1b